src/com/epam/javacourse/homework2/Palindrome.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.javacourse.homework2;
+
+public class Palindrome {
+    public boolean isPalindrome(int x) {
+        if (x < 0) {
+            return false;
+        }
+        int number = x;
+        int rev = 0;
+        int dig;
```

Comment:
This `dig` variable can be an inner loop variable, cause it's only used inside the loop.